src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts

```diff
@@ -2,7 +2,7 @@ import styled from '@emotion/styled'
 
 import { Pill } from '@/components/Pill'
 import { Text } from '@/components/Text'
-import { media, oldColors, sizes } from '@/styles'
+import { cVar, media, oldColors, sizes } from '@/styles'
```

Comment:
Sorry to bother you again 😅  but could we also replace all usages of `oldColors` to `cVar` in this file?

Reply:
it has been done 👍

Reply:
You removed `oldColors` in `VideoWorkspace.styles` and I was pointing to `VideoWorkspaceTabsBar.styles` 😅

Reply:
oopsy daisy, remove them as well now